utils/opencas.py

```diff
@@ -292,7 +292,7 @@ def check_cache_device_empty(self):
                 )
 
         def check_cache_mode_valid(self, cache_mode):
-            if cache_mode not in ['wt', 'pt', 'wa', 'wb', 'wo']:
+            if cache_mode not in ['wt', 'pt', 'wa', 'wb', 'wo', 'wi']:
```

Comment:
On the other hand when experimental features are disabled (which is default), WI should not be recognized as a valid cache mode. This option should be accepted only conditionally. The problem is that we currently do not have any good way to recognize whether OpenCAS was compiled with experimental features enabled, so this is another thing to be implemented in order to make `casctl` work with Write-Invalidate.

Reply:
Shouldn't, at some point, some C function fails if WI was not enabled during compile time? Making this patch "safe" even when WI is disabled? For example:
```
inline int validate_str_cache_mode(const char *s)
```